www/src/pages/en/usage/trpc.md

```diff
@@ -143,6 +144,91 @@ Compare this to the tRPC example above and you can see some of the advantages of
 - You don’t need to validate that the request query or body contains the correct data in the procedure, because Zod takes care of this.
 - Instead of creating a response, you can throw errors and return a value or object as you would in any other TypeScript function.
 
+## Useful snippets
+
+Here are some useful snippets that you might find useful for certain use cases.
```

Comment:
Between this line and the title above it, the word 'useful' is used three times. Maybe something like "Here are some snippets that might come in handy"? Or is that too informal?

Reply:
Haha you're right.